src/styling-panel-definition.js

```diff
@@ -11,7 +44,406 @@ const getStylingPanelDefinition = (bkgOptionsEnabled) => ({
       ref: 'components',
       useGeneral: true,
       useBackground: bkgOptionsEnabled,
+      items: {
+        labelSection: {
+          translation: 'Label',
+          component: 'panel-section',
+          show(data, handler, args) {
+            console.log("show=>labelSection=>", args)
+            const components = args?.properties?.components
+            if (!components) {
+              args.properties.components = []
+            }
+            // To convert the font style object to array, as the designs of font style is different in editor and panel
+            const styleArr = [];
+            Object.keys(args?.handler?.layout?.style?.font.style).forEach(styleName => {
+              if (args?.handler?.layout?.style?.font.style[styleName] === true) {
+                styleArr.push(styleName)
+              }
+            })
+
+            if (!args?.properties?.components?.find((comp) => comp.key === 'actionbutton')) {
+              const obj = { key: 'actionbutton' };
+
+              obj.font = {
+                'color': args?.handler?.layout?.style?.font.color,
+                'useColorExpression': args?.handler?.layout?.style?.font.useColorExpression,
+                'colorExpression': args?.handler?.layout?.style?.font.colorExpression,
+                'align': args?.handler?.layout?.style?.font.align,
+                'size': args?.handler?.layout?.style?.font.size,
+                'style': styleArr, // args?.handler?.layout?.style?.font.style
+              }
+              obj.border = args?.handler?.layout?.style?.border;
+              obj.bgColor = {
+                'color': args?.handler?.layout?.style?.background.color,
+                'colorExpression': args?.handler?.layout?.style?.background.colorExpression,
+                'useColorExpression': args?.handler?.layout?.style?.background.useColorExpression,
+              };
+              obj.bgImage = {
+                'mode': (args?.handler?.layout?.style?.background.useImage === true) ? 'media' : 'none',
+                'position': args?.handler?.layout?.style?.background.position,
+                'sizing': (args?.handler?.layout?.style?.background.size === 'auto') ? 'originalSize' : args?.handler?.layout?.style?.background.size,
+                'mediaUrl': {
+                  'qStaticContentUrl': {
+                    'qUrl': args?.handler?.layout?.style?.background.url.qStaticContentUrl.qUrl
+                  }
+                }
+              }
+              obj.icon = args?.handler?.layout?.style?.icon;
+              obj.label = args?.handler?.layout?.style?.label;
+              args.properties.components.push(obj)
+            }
+            return true;
+          },
+          items: {
+            labelItem: {
+              component: 'items',
+              ref: 'components',
+              key: 'actionbutton',
```

Comment:
I don't think these need to be in the components array. Only when this is something that could be generic it should be placed in components. The key `actionbutton` already shows that it is probably not generic